This shit is broken, gee.
On other platforms, this is usually implied but not here !
For some reasons, some OGG files would simply refuse to play without that file.
I think they fixed this in the unreleased SDL_mixer 1.2.13 though.

Gameblabla, 2018

diff -Nur SDL_mixer-1.2.12.orig/mixer.c SDL_mixer-1.2.12/mixer.c
--- SDL_mixer-1.2.12.orig/mixer.c
+++ SDL_mixer-1.2.12/mixer.c
@@ -476,6 +476,10 @@
 
 	audio_opened = 1;
 	SDL_PauseAudio(0);
+	
+	/* HACKKKKK RS-97 */
+	Mix_Init(MIX_INIT_OGG);
+	
 	return(0);
 }
 
